pandas/core/series.py

```diff
@@ -2702,8 +2705,10 @@ def cov(
         this, other = self.align(other, join="inner", copy=False)
         if len(this) == 0:
             return np.nan
+        this_values = np.asarray(this.values)
```

Comment:
Can we use ._values here?

Reply:
Yes I think so. For my knowledge, if we need an ndarray for `nanops.nancov`, is there a benefit to using `np.asarray(this._values)` vs `np.asarray(this.values)`?

Reply:
its probably harmless in this case but in general .values can be either a) lossy (dt64tz) or b) convert to object (period) while ._values is safe from those failure modes

Reply:
got it, thanks! I'll update just in case

Reply:
values also creates references and is read_only (in the future) for CoW. Better to use ._values